Scripts/developer_scripts/test_merge_of_branch

```diff
@@ -115,6 +117,15 @@ if [ -n "${project_name_demo}" ]; then
   exit 1
 fi
 
+#check no file contains non-utf8 characters
+echo '.. Checking if non utf-8 characters are used...'
+txt_not_utf8=$(for i in `find . -type f`; do out=`file $i`; if echo $out | grep -q "text"; then if ! (echo $out | egrep -q "UTF-8|ASCII|XML|EPS"); then echo $i; fi; fi; done)
```

Comment:
This loop is horrible! In my Git repository, including the build directories, I have 280440 files! For each file, at least one `file` and one `grep` is forked. That runs forever!